molgenis/molgenis-emx2
# chore:  load petstore via profile

### What are the main changes you did
- added the petstore demo schema, demo data and settings to the data folder 
- updated the dataModel to use the datafolder to load the petstore ( instaed of using the 'hard'coded petstore loader)

### How to test
- create a new schema using the ui , select template -> pet store , load-demo-data -> true 
- should load schema with data and settings as defined in the data folder 

### Checklist
